tf-environments/dev/terraform.tfvars

```diff
@@ -2,7 +2,7 @@ terragrunt = {
   remote_state {
     backend = "s3"
     config {
-      bucket     = "kubernetes-ops-1234-terraform-state"
+      bucket     = "kubernetes-ops-1234-kops-state-store"
```

Comment:
This actually should be the `terraform-state` store. We should keep the two state store between Terraform and Kops different since these are two different orchestration tools.  Don't want them to start overwritting each other.

Reply:
lemme change it back